### PR TITLE
Fix OpenSAML initialization error

### PR DESCRIPTION
**Pull request by mmoayyed**
A number of classes attempt to manually init opensaml.
All init functionality is handled via the opensaml config bean.

### CODE REVIEW

File: cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/util/AbstractSamlObjectBuilder.java [hunk 1]
```diff
@@ -100,22 +100,6 @@
     protected final Logger logger = LoggerFactory.getLogger(this.getClass());
 
     /**
-     * Initialize and bootstrap opensaml.
-     * Check for prior OpenSAML initialization to prevent double init
-     * that would overwrite existing OpenSAML configuration.
-     */
-    static {
```
Comment (leleuj) [inline cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/util/AbstractSamlObjectBuilder.java]: How is the initialization done in that case?

Comment (mmoayyed) [inline cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/util/AbstractSamlObjectBuilder.java]: It's triggered automatically via Spring and OpenSamlConfigBean, which does everything this component does and more.

Comment (mmoayyed) [inline cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/util/AbstractSamlObjectBuilder.java]: Also noteworthy that the above configuration causes CAS to crash

**Comment by leleuj** [comment]
+1

**Pull request merged**